src/Base/MultiFab.cpp

```diff
@@ -507,19 +507,31 @@ void init_MultiFab(py::module &m)
         .def("norm2", py::overload_cast< Vector<int> const & >(&MultiFab::norm2, py::const_))
 
         /* simple math */
+
         .def("sum",
              //py::overload_cast< int, bool >(&MultiFab::sum, py::const_),
              [](MultiFab const & mf, int comp , bool local) { return mf.sum(comp, local); },
```

Comment:
I also tried the following change (as a preparation to add another overload for `sum`), but it surprisingly fails to compile
```suggestion
        .def("sum",
             py::overload_cast< int, bool >(&MultiFab::sum, py::const_),
```

Reply:
That is probably some issue because the base class of MultiFab also defines `sum` overloads. We can simply use lambdas.